cypress/e2e/registration.cy.js

```diff
@@ -2,10 +2,67 @@
 
 describe('Student Registration page', () => {
   before(() => {
-
+    cy.visit('https://demoqa.com/automation-practice-form');
   });
+  const userName = 'Bohdan';
+  const userSurname = 'Svystun';
+  const email = 'randomemail@gmail.com';
+  const subject = 'No subjects';
+  it('should successfully register the user with valid credentials', () => {
+    cy.viewport(1920, 1080);
+
+    cy.get('[placeholder="First Name"]').type(userName);
+
+    cy.get('[placeholder="Last Name"]').type(userSurname);
+
+    cy.get('[placeholder="name@example.com"]').type(email);
+
+    cy.get('[for="gender-radio-1"]').click();
+
+    cy.get('[placeholder="Mobile Number"]').type('3809761337');
+
+    cy.get('[id="dateOfBirthInput"]').click();
+
+    cy.get('.react-datepicker__year-select').select('1991');
+
+    cy.get('.react-datepicker__day--028').click();
+
+    cy.get('[id="subjectsContainer"]').type(subject);
+
+    cy.get('[for="hobbies-checkbox-2"]').click();
+
+    cy.get('[placeholder="Current Address"]').type('Lviv');
+
+    cy.get('#state').type('Utt{enter}');
+
+    cy.get('#city').type('Ag{enter}');
+
+    cy.get('[id="submit"]').click();
+
+    cy.get('.modal-content').should('exist');
+
+    cy.get('.modal-header').should('contain', 'Thanks for submitting the form');
+
+    cy.get('.modal-body').should('contain', 'Bohdan');
+
+    cy.get('.modal-body').should('contain', 'Svystun');
+
+    cy.get('.modal-body').should('contain', 'email');
+
+    cy.get('.modal-body').should('contain', 'Gender', 'male');
+
+    cy.get('.modal-body').should('contain', 'Mobile', '3809761337');
+
+    cy.get('.modal-body').should('contain', 'Date of Birth', '28 September');
```

Comment:
In the modal body assertion for 'Date of Birth', '28 September' is hardcoded. It would be better to assert the date selected during registration.